src/main/resources/i18n/messages.properties

```diff
@@ -281,4 +281,8 @@ lang.export.field-order=Mezők sorrendje
 lang.export.pdf=PDF
 lang.export.csv=CSV
 lang.export.scheme=Séma kiválasztása:
+lang.export.page_orientation=Oldalbeállítás:
+lang.export.landscape=Fekvő
+lang.export.portrait=Álló
+lang.export.empty_rows=Üres sorok hozzáadása:
```

Comment:
Just to be consistent, please replace the underscore with dashes.

Reply:
Applied all your suggestions, thank you for the tips and tricks, all very useful!